frontend/src/metabase/browse/components/BrowseModels.styled.tsx

```diff
@@ -82,3 +83,23 @@ export const CollectionHeaderGroup = styled(Group)`
   position: relative;
   top: 0.5rem;
 `;
+
+export const Banner = styled.div`
```

Comment:
I think a lot of these styled components can mostly be done with Mantine components instead. `Banner` could be a `<Flex>` with some style props added, and we could grow the text so that it pushes the close button to the right.